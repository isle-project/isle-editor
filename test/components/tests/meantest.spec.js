// MODULES //

import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import MeanTest from '@isle-project/components/tests/meantest';
import heartdisease from '../fixtures/heartdisease.json';


// TESTS //

describe( '<MeanTest />', function test() {
	it( 'renders test result', () => {
		const output = renderer.create( <MeanTest
			data={heartdisease}
			variable="Drugs"
		/> );
		expect( output ).toMatchSnapshot();
	});
});
