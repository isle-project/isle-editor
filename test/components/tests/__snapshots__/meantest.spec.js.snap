// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MeanTest /> renders test result 1`] = `
<div
  style={
    {
      "overflowX": "auto",
      "width": "100%",
    }
  }
>
  <label>
    hypothesis-test-for
    :
  </label>
  <div />
  <pre>
    One-sample t-test

Alternative hypothesis: True mean is not equal to 0

    pValue: 0
    statistic: 14.1712
    df: 787
    95% confidence interval: [0.2919,0.3858]


  </pre>
</div>
`;
